package.json

```diff
@@ -71,12 +71,13 @@
     "@types/jest": "^27.5.1",
     "@types/lodash": "^4.14.176",
     "@types/node": "^18.18.5",
-    "@types/react": "^16.14.20",
+    "@types/react": "16.14.40",
     "@types/react-dom": "^16.9.14",
     "@types/react-resizable": "^1.7.4",
     "@types/react-router": "^5.1.18",
     "@types/react-router-dom": "^5.3.2",
     "@types/react-transition-group": "^4.4.4",
+    "@types/trusted-types": "^2.0.7",
```

Comment:
This dependency is problematic. It is supposed to be a bult-in `dom.d.ts` type, but temporarily it exists as a library.

When it becomes built-in, our consumers may have problems upgrading to the latest typescript because of that. 

Is type-safety here really that much important, or can we get away with type casting

```js
iconSvg: trustedPolicy.createHTML(svgHtml) as any as string,
```